jfrog-applications/ide/jetbrains-ides/plugin-configuration.md

```diff
@@ -0,0 +1,25 @@
+# Plugin Configuration
+
+To access the plugin configuration, click on the gear icon:
+
+
+![setting-button](../../.gitbook/assets/intellij/settingButton.png)
+
+## Advanced
+
+#### Downloading External Resources Through Artifactory
+
+JFrog IntelliJ Plugin requires necessary resources for scanning your projects.
+By default, the JFrog IntelliJ Plugin downloads the necessary resources needed from https://releases.jfrog.io. If the machine that runs JFrog IntelliJ Plugin has no access to it, follow these steps to allow the tools to be downloaded through an Artifactory instance, which the machine has access to:
+
+1. Login to the JFrog Platform UI, with a user who has admin credentials.
+
+2. Create a Remote Repository with the following properties set:
+    - Under the 'Basic' tab:
+        Package Type: Generic
+        URL: https://releases.jfrog.io
+    - Under the 'Advanced' tab:
+        Uncheck the 'Store Artifacts Locally' option
```

Comment:
The new-lines in the rows above become invisible in the markdown view.
![image](https://github.com/jfrog/documentation/assets/7830056/6f8599b6-7ca1-4f94-a905-ce0459cddebd)
Also, let's add the relevant screenshots from the Platform UI here.